diff --git a/drivers/acpi/resource.c b/drivers/acpi/resource.c
index 3d74ebe9d..b4da6fd47 100644
--- a/drivers/acpi/resource.c
+++ b/drivers/acpi/resource.c
@@ -559,6 +559,13 @@ static const struct dmi_system_id irq1_level_low_skip_override[] = {
 			DMI_MATCH(DMI_BOARD_NAME, "17U70P"),
 		},
 	},
+	{
+		/* LG Electronics 17U70P */
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Standard"),
+			DMI_MATCH(DMI_BOARD_NAME, "Standard"),
+		},
+	},
 	{ }
 };
 
@@ -695,6 +702,12 @@ static const struct dmi_system_id irq1_edge_low_force_override[] = {
 			DMI_MATCH(DMI_BOARD_NAME, "GMxHGxx"),
 		},
 	},
+	{
+		/* TongFang GMxHGxx/TUXEDO Stellaris Slim Gen1 AMD */
+		.matches = {
+			DMI_MATCH(DMI_BOARD_NAME, "Standard"),
+		},
+	},
 	{ }
 };
 
